build.yaml

```diff
@@ -934,3 +949,18 @@ job_type:
                    *build_docker_image,
                    *push_image_to_dockerhub ]
           }
+    build_vagrant_basebox_for_flocker_tutorial:
+      at: '0 2 * * *'
+      on_nodes_with_labels: 'mesos-512MB'
+      with_steps:
+        - { type: 'shell',
+            cli: [ *add_shell_functions, *setup_pip_cache,
+                   *cleanup, *setup_venv, *setup_flocker_modules,
+                   *check_version,
+                   'export DISTRIBUTION_NAME=centos-7',
+                   *build_sdist, *build_package,
+                   *build_repo_metadata,
+                   'export HOME=/root',
+                   *build_vagrant_tutorial_basebox,
+                   *clean_packages]
+          }
```

Comment:
we only build the master branch once every 24hours, all other branches can be built on demand as needed